db.ngo.us

```diff
@@ -1,75 +1,4 @@
-$ORIGIN ngo.us.
-$TTL 86400
-; SOA Records
-@		IN	SOA	ns2.afraid.org. admin.nic.ngo.us. YYYYMMDDNN 86400 10800 3600000 3600
-
-; ========== BEGIN REGISTRY SERVICES ==========
-
-; Delegation Records
-@		IN	NS	ns1.alstra.org.
-@		IN	NS	ns2.alstra.org.
-@		IN	NS	ns3.alstra.org.
-@		IN	NS	ns2.afraid.org.
-@		IN	NS	ns1.1984hosting.com.
-
-; NS Records
-client		IN	NS	paloma.ns.cloudflare.com.
-client		IN	NS	sergi.ns.cloudflare.com.
-nic	86400	IN	NS	paloma.ns.cloudflare.com.
-nic	86400	IN	NS	sergi.ns.cloudflare.com.
-
-; A Records
-@	3600	IN	A	3.5.130.81
-whois	60	IN	A	192.0.2.1
-
-; TXT Records
-_psl		IN	TXT	https://github.com/publicsuffix/list/pull/1821
-_whois._tcp	86400	IN	TXT	whois.ngo.us
-
-; Others
-_whois._tcp	3600	IN	SRV	0 0 43 whois.ngo.us.
-
-; ========== END REGISTRY SERVICES ============
-
-; ========== BEGIN ORGANIZATIONS ==============
-
-; Lively Souls Foundation
-liveleak	86400	IN	NS	dns1.p08.nsone.net.
-liveleak	86400	IN	NS	dns2.p08.nsone.net.
-liveleak	86400	IN	NS	dns3.p08.nsone.net.
-liveleak	86400	IN	NS	dns4.p08.nsone.net.
-opentube	86400	IN	NS	dns1.p08.nsone.net.
-opentube	86400	IN	NS	dns2.p08.nsone.net.
-opentube	86400	IN	NS	dns3.p08.nsone.net.
-opentube	86400	IN	NS	dns4.p08.nsone.net.
-pics	86400	IN	NS	dns2.p08.nsone.net.
-pics	86400	IN	NS	dns3.p08.nsone.net.
-
-; PAUSE International
-; Submitted by PAUSE International <pauseint@proton.me>
-pause	86400	IN	NS	dns1.wndp.pt.
-pause	86400	IN	NS	dns2.wndp.pt.
-pauseint	86400	IN	NS	dns1.wndp.pt.
-pauseint	86400	IN	NS	dns2.wndp.pt.
-
-; Project CN-Archive
-; Submitted by WPD <wpdaily.websites@skiff.com>
-ngocn-news-archive	86400	IN	NS	dns1.freehostia.com.
-ngocn-news-archive	86400	IN	NS	dns2.freehostia.com.
-
-; Sea Food Distribution
-health	86400	IN	NS	ns1.he.net.
-health	86400	IN	NS	ns2.he.net.
-health	86400	IN	NS	ns3.he.net.
-health	86400	IN	NS	ns4.he.net.
-
-; YMHC Charitable Foundation
-; Submitted by ITS <its@ymhc.ngo>
-schoolexclusionstories	86400	IN	NS	achiel.ns.cloudflare.com.
-schoolexclusionstories	86400	IN	NS	dolly.ns.cloudflare.com.
-schoolphobia	86400	IN	NS	achiel.ns.cloudflare.com.
-schoolphobia	86400	IN	NS	dolly.ns.cloudflare.com.
-ymhc	86400	IN	NS	achiel.ns.cloudflare.com.
-ymhc	86400	IN	NS	dolly.ns.cloudflare.com.
-
-; ========== END ORGANIZATIONS ================
+; Vihanga Ashinshana Projects
```

Comment:
Please insert your organization's records. Ensure the domain blocks are ordered alphabetically by your organization's name. Do not delete the rest of the zone file or modify lines that are not associated with your organization.